components/Navigation.css

```diff
@@ -0,0 +1,33 @@
+body {
+    background-color : lightgray;
+   
```

Comment:
remove blank spaces and give proper alignment before merging
you can use prettier or beautify for better ease.